src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixLegacyResourceDesignerStep.cs

```diff
@@ -68,18 +70,20 @@ protected override void LoadDesigner ()
 			}
 		}
 
-		internal override bool ProcessAssemblyDesigner (AssemblyDefinition assembly)
+		internal override bool ProcessAssemblyDesigner (AssemblyDefinition assembly, TypeDefinition designer = null)
 		{
-			if (!FindResourceDesigner (assembly, mainApplication: false, out TypeDefinition designer, out CustomAttribute designerAttribute)) {
+			if (designer is not null) {
+				LogMessage ($"   {assembly.Name.Name} has an assembly reference with a designer.");
+			} else if (!FindResourceDesigner (assembly, mainApplication: false, out designer, out _)) {
```

Comment:
This is hitting "perfect is the enemy of the good" territory -- so don't take this comment as a suggestion to change anything -- but I don't like the code flow here:

  * It requires that `FindResourceDesigner` create a new `TypeDefinition` instance, even though that instance shouldn't be *used* for anything, and ideally should be linked away entirely.
  * I suspect we could have/create a scenario in which [an assembly references *multiple* `Resource` types](https://discord.com/channels/732297728826277939/732297837953679412/1163452800643104799), in which case returning one type -- if that type were *used* for anything -- would be inappropriate.

This complaint likely isn't worth addressing *now*, I just wanted to comment on it.